src/Uno.UI.FluentTheme/Resources/PriorityDefault/CommandBar_themeresources.xaml

```diff
@@ -1,1716 +1,2668 @@
-            <x:Double x:Key="CommandBarOverflowMaxHeight">198</x:Double>
-            <Thickness x:Key="CommandBarOverflowPresenterBorderThickness">1</Thickness>
-            <Thickness x:Key="CommandBarOverflowPresenterBorderDownThickness">0,0,0,1</Thickness>
-            <Thickness x:Key="CommandBarOverflowPresenterBorderUpThickness">0,1,0,0</Thickness>
-            <Thickness x:Key="CommandBarOverflowPresenterBorderPadding">0</Thickness>
-            <Thickness x:Key="CommandBarOverflowPresenterBorderDownPadding">0</Thickness>
-            <Thickness x:Key="CommandBarOverflowPresenterBorderUpPadding">0</Thickness>
-        </ResourceDictionary>
-        <ResourceDictionary x:Key="Light">
-            <x:Double x:Key="CommandBarOverflowMinWidth">160</x:Double>
-            <x:Double x:Key="CommandBarOverflowTouchMinWidth">240</x:Double>
-            <x:Double x:Key="CommandBarOverflowMaxWidth">480</x:Double>
-            <x:Double x:Key="CommandBarOverflowMaxHeight">198</x:Double>
-            <StaticResource x:Key="CommandBarBackground" ResourceKey="SystemControlBackgroundChromeMediumBrush" />
-            <StaticResource x:Key="CommandBarForeground" ResourceKey="SystemControlForegroundBaseHighBrush" />
-            <StaticResource x:Key="CommandBarHighContrastBorder" ResourceKey="SystemControlForegroundTransparentBrush" />
-            <StaticResource x:Key="CommandBarEllipsisIconForegroundDisabled" ResourceKey="SystemControlDisabledBaseMediumLowBrush" />
-            <StaticResource x:Key="CommandBarOverflowPresenterBackground" ResourceKey="SystemControlTransientBackgroundBrush" />
-            <StaticResource x:Key="CommandBarOverflowPresenterBorderBrush" ResourceKey="SystemControlTransientBorderBrush" />
-            <StaticResource x:Key="CommandBarLightDismissOverlayBackground" ResourceKey="SystemControlPageBackgroundMediumAltMediumBrush" />
-            <Thickness x:Key="CommandBarOverflowPresenterBorderThickness">1</Thickness>
-            <Thickness x:Key="CommandBarOverflowPresenterBorderDownThickness">0,0,0,1</Thickness>
-            <Thickness x:Key="CommandBarOverflowPresenterBorderUpThickness">0,1,0,0</Thickness>
-            <Thickness x:Key="CommandBarOverflowPresenterBorderPadding">0</Thickness>
-            <Thickness x:Key="CommandBarOverflowPresenterBorderDownPadding">0</Thickness>
-            <Thickness x:Key="CommandBarOverflowPresenterBorderUpPadding">0</Thickness>
-        </ResourceDictionary>
-    </ResourceDictionary.ThemeDictionaries>
+					xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
+	<ResourceDictionary.ThemeDictionaries>
+		<ResourceDictionary x:Key="Default">
+			<x:Double x:Key="CommandBarOverflowMinWidth">160</x:Double>
+			<x:Double x:Key="CommandBarOverflowTouchMinWidth">240</x:Double>
+			<x:Double x:Key="CommandBarOverflowMaxWidth">480</x:Double>
+			<x:Double x:Key="CommandBarOverflowMaxHeight">198</x:Double>
+			<StaticResource x:Key="CommandBarBackground"
```

Comment:
try not to change the white spacing in Uno.UI.FluentTheme resources, since it makes it harder to manage upstream changes from the MUX repo.

It also makes it impossible to see what actual functional changes you've made here... whatever they are, prefix them with a big `<!-- UNO WORKAROUND: [Explanation] -->` comment, or else they'll get lost the first time somebody copies in the latest version from upstream.